R-package/NAMESPACE

```diff
@@ -10,10 +10,8 @@ S3method(getinfo,xgb.Booster)
 S3method(getinfo,xgb.DMatrix)
 S3method(length,xgb.Booster)
 S3method(predict,xgb.Booster)
-S3method(print,xgb.Booster)
```

Comment:
I think this will break existing behaviors

Reply:
This is part of the work on the new R interface: https://github.com/dmlc/xgboost/issues/9810

Reply:
It shouldn't be related to the changes in other PRs. In the current master branch, triggering docs build with roxygen doesn't remove the S3 method exports.

Reply:
I have now added the roxygen tags @method print xgboost and @method print xgb.Booster. This brings the NAMESPACE file back to its original state.